pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -180,6 +183,25 @@ func (d *dpiComponent) dpiDaemonset() *appsv1.DaemonSet {
 	if d.cfg.TyphaNodeTLS.NodeSecret.UseCertificateManagement() {
 		initContainers = append(initContainers, d.cfg.TyphaNodeTLS.NodeSecret.InitContainer(DeepPacketInspectionNamespace))
 	}
+	if d.cfg.IntrusionDetection.Spec.DeepPacketInspectionDaemonset.SnortInitContainers != nil {
+		for iter, snortInitContainer := range d.cfg.IntrusionDetection.Spec.DeepPacketInspectionDaemonset.SnortInitContainers {
+			container := corev1.Container{
+				Name:            fmt.Sprintf("%s-%d", DeepPacketInspectionInitContainerName, iter),
+				Image:           snortInitContainer.Image,
+				ImagePullPolicy: corev1.PullAlways,
```

Comment:
We shouldn't leave the pull policy as always. We have a helper function for this: https://github.com/tigera/operator/blob/master/pkg/render/intrusion_detection.go#L659

Reply:
Corrected.